main_routers/system_router.py

```
@@ -0,0 +1,483 @@
+        logger.error(f"情绪分析失败: {e}")
+        return JSONResponse({"success": False, "error": str(e)}, status_code=500)
+
+
+@router.post('/api/steam/set-achievement-status/{name}')
+async def set_achievement_status(name: str):
+    """设置Steam成就状态"""
+    steamworks = get_steamworks()
+    
+    if steamworks is not None:
+        try:
+            # 先请求统计数据并运行回调，确保数据已加载
```

Ruff (0.14.8)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)